cmd/dep/main.go

```diff
@@ -153,6 +166,26 @@ func (c *Config) Run() int {
 		return successExitCode
 	}
 
+	// 'dep help constraints' generates documentation on the exact syntax for version constraints
+	if printCommandHelp && cmdName == "constraints" {
+		fmt.Println("Version constraint syntax:")
+		fmt.Println()
+		firstSentence := `"dep ensure" uses an external semver (https://github.com/Masterminds/semver) `
+		firstSentence += "library to interpret the version constraints you specify in the manifest."
```

Comment:
This line length is 80+, let's divide it into multiple lines with length under 80 characters. You could use multiline backticks.